File: 0001_Two_sum/Brute_force.java

public class Brute_force {
    public static int[] twoSum(int[] nums, int target){
        int[] res = null;
        for (int i = 0; i < nums.length; i++) {
            for(int j = 0; j < nums.length; j++){
                if((i != j) && (nums[i] + nums[j] == target)){
                    res = new int[]{i, j};
                }
            }
        }
        return res;
    }
}

File: 0001_Two_sum/Brute_force.js
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function (nums, target) {
    for (let i = 0; i < nums.length; i++) {
        for (let pointer = 0; pointer < nums.length; pointer++) {
            if (i != pointer && (nums[i] + nums[pointer]) == target) {
                return [i, pointer]
            }
        }
    }
    return null
};